var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -69,6 +69,12 @@ def install(self, spec, prefix):
             '--with-blas-libs=%s' % ' '.join(blas.names),
             '--with-blas-lib-dirs=%s' % ' '.join(blas.directories)
         ]
+        
+        if spec.satisfies('^netlib-lapack'):
+            # netlib-lapack puts the libs in lib64
+            configure_args.extend([
+                "--with-lapack-lib-dirs=%s/lib64".join(lapack.directories),
```

Comment:
@eklee15 look at https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/netlib-lapack/package.py#L73-L85

it at looks good to me! So you need to debug and see what value do you get for `lapack.directories` and `blas.directories` and why `hypre` does not like it. It all should be working OOB.